catalogs/catalog-hive/src/test/java/org/apache/gravitino/catalog/hive/TestHiveTableOperations.java

```diff
@@ -31,8 +31,8 @@
 import java.util.Arrays;
 import java.util.Map;
 import org.apache.gravitino.NameIdentifier;
-import org.apache.gravitino.catalog.hive.miniHMS.MiniHiveMetastoreService;
 import org.apache.gravitino.exceptions.NoSuchPartitionException;
+import org.apache.gravitino.hms.miniHMS.MiniHiveMetastoreService;
```

Comment:
Can we change the package name to "o.a.g.hive.hms.xxx", it is weird that the package name has uppercase letters...

Reply:
renamed